include/swift/AST/DiagnosticsSema.def

```diff
@@ -3102,6 +3102,11 @@ ERROR(invalid_weak_ownership_not_optional,none,
       "'weak' variable should have optional type %0", (Type))
 ERROR(invalid_weak_let,none,
       "'weak' must be a mutable variable, because it may change at runtime", ())
+ERROR(protocol_property_declarations_cannot_specify_ownership,none,
+      "ownership cannot be specified in protocols", ())
```

Comment:
I would prefer if the diagnostic mentioned the keyword in question, like "'weak' properties are not supported in protocols"

Reply:
I think Greg's wording is better, actually. This way people won't be concerned about using a weak property to *satisfy* a requirement.

Reply:
(which, admittedly, might be dangerous, but let's say `unowned` instead)

Reply:
Would the following be clearer error messages? 

```ownership keyword 'weak' is not supported in protocols```
```ownership keyword 'unowned' is not supported in protocols```

Reply:
@gspiers My meta-suggestion is to look for existing diagnostics concerning ownership keywords, and try to make the new ones consistent with the existing ones.